install.sh

```diff
@@ -66,7 +66,7 @@ fi
 echo "Building Wayfire $STREAM"
 echo "Installation prefix: $PREFIX"
 
-BUILDROOT=$(pwd)
+BUILDROOT="${0%/*}"
```

Comment:
What does this do?

Reply:
If for instance you clone wf-install and then run `./wf-install/install.sh` then $(pwd) will be `.` and when it goes to find files from the build root, specifically the start script .in, build root will not be the repo directory, so it doesn't find the file. The 0 in "${0%/*}" would be `./wf-install/install.sh` and it eliminates everything after and including the last forward slash. So it would evaluate to `./wf-install`. This works regardless which directory it is run from.

Reply:
-1, the user should go where they want to build Wayfire.